oracle/config/samples/v1alpha1_config_kind.yaml

```diff
@@ -7,7 +7,7 @@ spec:
   storageClass: "csi-hostpath-sc"
   volumeSnapshotClass: "csi-hostpath-snapclass"
   images:
-    service: "gcr.io/oracle-database-images/oracle-12.2-ee-seeded-mydb:kind"
-    dbinit: "gcr.io/oracle.db.anthosapis.com/dbinit:kind"
-    logging_sidecar: "gcr.io/oracle.db.anthosapis.com/loggingsidecar:kind"
-    monitoring: "gcr.io/oracle.db.anthosapis.com/monitoring:kind"
+    service: "localhost:5000/oracle-12.2-ee-seeded-mydb:latest"
```

Comment:
Can you please add a comment indicating that localhost:5000 is the default address for a docker registry?

Reply:
Oops, merged before seeing this comment.

This sample yaml file is referred in the kind user guide here: https://github.com/GoogleCloudPlatform/elcarro-oracle-operator/blob/main/docs/content/kind.md#creating-a-kind-config-cr. And in that user guide, we've using "localhost:5000" all the places for local docker registry. Do we still need the comment in this yaml file in this case?